substrate-node/pallets/pallet-tfgrid/src/types.rs

```diff
@@ -176,7 +176,10 @@ impl<B: Ord> PartialOrd for FarmingPolicy<B> {
 impl<B: Ord> Ord for FarmingPolicy<B> {
     fn cmp(&self, other: &Self) -> Ordering {
         match self.farm_certification.cmp(&other.farm_certification) {
-            Ordering::Equal => self.node_certification.cmp(&other.node_certification),
+            Ordering::Equal => match self.node_certification.cmp(&other.node_certification) {
```

Comment:
this `cmp()` is set for being able to use `sort()` on  `FarmingPolicy`, correct?
For our usage we only care about the creation time so why `id` ordering is not enough?

Reply:
Sorting by ID will always make sure policies are sorted by creation time since ID is incremental.